rails/app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Terrastories</title>
+    <title>Welcome - Terrastories<%= @community ? ': ' + @community.name : '' %></title>
```

Comment:
Let's use interpolation rather than string concatenation and check for community name in addition to community existence:

```suggestion
    <title>Welcome - Terrastories<%= @community&.name ? ": #{@community.name}": '' %></title>
```

In particular, this will ensure that if community exists but name is somehow nil, it won't crash the page.

```shell
irb> ': ' + nil
TypeError (no implicit conversion of nil into String)
irb> ": #{nil}"
=> ": "
```

@rudokemper Do we also want to localize "Welcome" ?

Reply:
Great suggestions @lauramosher and yes - that would be a welcome :facepalm: change.

Reply:
Pushed the nil safe and interpolation update, as well as localization!